slide/src/test/ui/extra_items.slide

```diff
@@ -12,6 +12,7 @@ error[P0001]: Unexpected extra tokens
   |
 1 |   1 + 0 -1 2 3
   |  __________^
+  |            - help: if you meant to specify another statement, add a newline before this token
 2 | |   4 5 6 / 7 ^ 8
```

Comment:
i don't think these tests are consitent for example. line 14 has a bunch of ___ before the ^ , but the test in tokens_past_expr has 3 ^ values and no _

Reply:
because the diagnostic in this test spans to line 3. Does that make sense?